node/src/chain_spec/oak.rs

```diff
@@ -13,8 +13,11 @@ use primitives::{AccountId, AuraId, Balance};
 
 const TOKEN_SYMBOL: &str = "OAK";
 const SS_58_FORMAT: u32 = 51;
+const STAGING_SS_58_FORMAT: u32 = 52;
```

Comment:
i know we need to register our ss58 somewhere for this to work. but we need something different from 51 i think?

Reply:
nope, we can use 51

Reply:
51 is an address format. We'd like the address format to be the same across our chains: staging, turing, oak

Reply:
even if turing staging with ss58 51 already exists in rococo?

Reply:
it's not. For example, mangatax uses the substrate default ss58 address. So do many of the parachains

Reply:
awesome, saves a bunch of work. will use the same 51 ss58!